### PR TITLE
Add trad/salt client comparison table

### PR DESCRIPTION
**Pull request by Loquacity**
Original: https://www.suse.com/documentation/suse-manager-3/3.2/susemanager-best-practices/html/book.suma.best.practices/mgr.product.comparison.html

### CODE REVIEW

**Comment by Loquacity** [comment]
image

File: modules/client-configuration/pages/client-config-overview.adoc [hunk 1]
```diff
@@ -6,3 +6,44 @@ Registering clients is the first step after installing {productname}, and most o
...
+In this table, icon:check[role="green"] indicates that the feature is available, and icon:times[role="danger"] indicates that it is not available.
+Fields marked as icon:question[role="gray"] are under consideration, and may or may not be made available at a later date.
+
+[cols="1,1,1,1", options="header"]
+.Features of Traditional and Salt Clients
+|===
+| Feature               | Notes | Traditional               | Salt
+| Registration          |       | icon:check[role="green"]  | icon:check[role="green"]
+| Install Packages      |       | icon:check[role="green"]  | icon:check[role="green"]
+| Install Patches       |       | icon:check[role="green"]  | icon:check[role="green"]
```
Comment (moio) [inline modules/client-configuration/pages/client-config-overview.adoc]: Usual jargon is to *apply* patches.

File: modules/client-configuration/pages/client-config-overview.adoc [hunk 2]
```diff
@@ -6,3 +6,44 @@ Registering clients is the first step after installing {productname}, and most o
...
+| Remote Commands       |       | icon:check[role="green"]  | icon:check[role="green"]
+| System Package States |       | icon:times[role="danger"] | icon:check[role="green"]
+| System Custom States  |       | icon:times[role="danger"] | icon:check[role="green"]
+| Group Custom States   |       | icon:times[role="danger"] | icon:check[role="green"]
+| Organization Custom States |  | icon:times[role="danger"] | icon:check[role="green"]
+| System Set Manager    |       | icon:check[role="green"]  | icon:check[role="green"]
+| Service Pack Migration |      | icon:check[role="green"]  | icon:check[role="green"]
+| Virtual Host Management | Bootstrap installation support | icon:check[role="green"] | icon:check[role="green"] (read-only)
```
Comment (moio) [inline modules/client-configuration/pages/client-config-overview.adoc]:
```suggestion
| Virtual Host Management | icon:check[role="green"] | icon:check[role="green"]
| Virtual Guest installation | icon:check[role="green"] | icon:question[role="gray"]
```

File: modules/client-configuration/pages/client-config-overview.adoc [hunk 3]
```diff
@@ -6,3 +6,44 @@ Registering clients is the first step after installing {productname}, and most o
...
+| System Redeployment | With Auto-installation | icon:check[role="green"] | icon:question[role="gray"]
+| Contact Methods | Between server and client | osad, rhnsd, ssh push | zeromq: Salt default salt-ssh
```
Comment (moio) [inline modules/client-configuration/pages/client-config-overview.adoc]:
```suggestion
| Contact Methods | Between server and client | osad, rhnsd, ssh push | zeromq (Salt default), salt-ssh
```

File: modules/client-configuration/pages/client-config-overview.adoc [hunk 4]
```diff
@@ -6,3 +6,44 @@ Registering clients is the first step after installing {productname}, and most o
...
+| Red Hat Network Clients |     | icon:check[role="green"]  | icon:check[role="green"]
```
Comment (moio) [inline modules/client-configuration/pages/client-config-overview.adoc]: I do not know what this means. Is this meant to be "RHEL/CentOS/Fedora/derivatives"?

Comment (Loquacity) [reply modules/client-configuration/pages/client-config-overview.adoc]: I assumed it was clients running Red Hat operating systems, which is why I added in the Ubuntu clients one beneath it.

Comment (moio) [reply modules/client-configuration/pages/client-config-overview.adoc]: [Red Hat Network](https://en.wikipedia.org/wiki/Red_Hat_Network) is the project that became Spacewalk, I do not think it is an appropriate name in this context.

We should probably list OSs explicitly - technically CentOS and Fedora are not Red Hat OSs either.

AFAIK, Red Hat Enterprise Linux 6 and 7 and SUSE Linux Enterprise Expanded Support are officially supported as clients. CentOS 6 and 7, Scientific Linux, Oracle Linux and other RHEL derivatives are known to work but not officially supported. Ubuntu LTS 16.04 LTS and 18.04 LTS are officially supported. Other versions, Debian, and other Debian derivatives might also work but they are not officially supported.

Comment (Loquacity) [reply modules/client-configuration/pages/client-config-overview.adoc]: Considering we already have a supported clients matrix, I'd rather not replicate it here. Let's just remove those lines.

**Comment by Loquacity** [comment]
image

**Pull request merged**